03/app.js

```diff
@@ -0,0 +1,49 @@
+const input = document.querySelector('input');
+const pEl = document.createElement('p');
+
+
+input.addEventListener('change', showImg);
+
+function showImg(e){
+    const file = [...e.target.files];
+    
+    
+    file.forEach(function(el){
+        if (el && el.type.includes('image')) {
+            addImg(el);
+        } else {
+            wrongFile(el);
+        };
+    })
+}
+
+
+function addImg(element) {
+    pEl.innerText = "";
+    const ulList = document.querySelector('.images-list');
+    const liEl = document.querySelector('.images-list__item--prototype');
+    const clone = liEl.cloneNode(true);
+    clone.classList.remove('images-list__item--prototype');
+    const liHeader = clone.querySelector('.images-list__item-name');
+    const liImage = clone.querySelector('.images-list__item-img');
+    const liFooter = clone.querySelector('.images-list__item-size');
```

Comment:
Może warto zamienić to na dodatkową funkcję tj. `createNewElement()` - będzie czytelniej ;)